docs/configuration.adoc

```diff
@@ -68,9 +69,19 @@ Components Registry is originally stored as configuration files with ConfigSlurp
 
 Each Component must be configured in corresponding configuration file according to following format:
 ----
-<component name> {
-        buildSystem = <build tool> // MAVEN, GRADLE etc. (default for all versions)
+<component id> {
+        componentDisplayName = "Name of component" //  DMS description
+        parentComponent      = <component id> // grouping components in DMS
+        clientCode           = <client code> // if component is client specific
+        solution             = <true|false> // true - meta-component
+
+        componentOwner   = "username of component's owner" //without domain part. For example: salekseev
+        releaseManager   = "username of component's release manager" 
+        securityChampion = "username of component's security officer" 
 
+        buildSystem = <build tool> // MAVEN, GRADLE etc. (default for all versions)
+        system      = <{system_name}>
```

Comment:
Please, comment system field

Reply:
// used for differentiation between different corporate policies (license control, etc)

Reply:
// used for devide corporate policies (license control, etc)